app/services/serializer_service.rb

```diff
@@ -321,13 +323,28 @@ def self.execute_query(operation_name, variables)
       number
       title
       description
+      tags
+      zones
       datePublication
       service { nom organisme typeOrganisme }
-      cadreJuridique
-      deliberation
+      lienDemarche
+      lienDpo
+      lienNotice
+      lienSiteWeb
+      lienCadreJuridique
```

Comment:
pas trop fan du nommage `lienX` , j'aurais privilégié `XUrl` (ou à la limite `UrlX` pour les garder groupées) mais c'est peut-être par soucis d'homogénéité avec d'autres champs ailleurs